### DISCUSSION TITLE
A Place for Golden Bonsai

### DISCUSSION THREAD

**Discussion by SirJoreee** [sunflower-land/sunflower-land#1205] [Art] [answered]
i have some bonsais at home and i cant keep but notice ingame its just lying on the ground.

I would like to suggest a good place for it or even just an additive like a table or pedestal of sort.

Take for example the one beside shop
*edited pic credits to @sircanada

it's a lot nicer 💛
![IMG_20220710_144444](https://user-images.githubusercontent.com/104824906/178135235-98f5bda3-087d-4c1e-8296-79be5c6efc14.jpg)
![unknown-1](https://user-images.githubusercontent.com/104824906/178135241-99e97bb5-a5dd-4572-8584-dc770257b455.png)

**Comment by adamhannigan** [comment, marked as answer]
@Joreee Nice suggestion! I'm happy for a developer to pick this one up.

**Reply by veenhdek** [reply]
I'll pick this up! Good intro to pixel art changes.

**Reply by SirJoreee** [reply]
Thank you Adam! Looking forward to your work veenhdek 🧡

**Reply by veenhdek** [reply]
> Thank you Adam! Looking forward to your work veenhdek 🧡

Just got merged! Great suggestion!
